pyproject.toml

```diff
@@ -100,6 +145,14 @@ known_first_party = ["lilio"]
 src_paths = ["lilio", "tests"]
 line_length = 120
 
+[tool.flake8]
+max-line-length = 88
+ignore = [
+  "E203",  # Whitespace before ":". Not PEP8 compliant (https://github.com/psf/black/issues/315)
+  "E501",  # Line length: currently (some) docstrings do not comply.
+  "W503",  # https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator
```

Comment:
Not sure why we want to silence these. Can we make it warn but not fail? I do think we should address them at some point.

Reply:
We need to silence E203 and W503. 

- E203 will conflict with black, and does not follow PEP8. 
- W503 also has to be silenced, otherwise you cannot split `a and b` over multiple lines. The PEP8 way to do this to have a line break before a binary operator (see the hyperlink).

AFAIK there is no way to turn a flake8 error into a warning. And if we make flake8 not fail the linter it will be ignored.

Reply:
Okay, can we then insert a link to https://black.readthedocs.io/en/stable/faq.html#why-are-flake8-s-e203-and-w503-violated?